src/test/resources/docker/4.0/Dockerfile

```diff
@@ -7,6 +7,6 @@ COPY target/datastax-mcac-agent-*.jar /opt/mcac/lib/datastax-mcac-agent.jar
 COPY target/collectd /opt/mcac/lib/collectd
 COPY config/collectd.conf.tmpl /opt/mcac/config
 
-RUN echo "JVM_OPTS=\"\$JVM_OPTS -javaagent:/opt/mcac/lib/datastax-mcac-agent.jar\"" >> /etc/cassandra/cassandra-env.sh
+RUN echo "JVM_OPTS=\"\$JVM_OPTS -javaagent:/opt/mcac/lib/datastax-mcac-agent.jar -Dcassandra.ring_delay_ms=0\"" >> /etc/cassandra/cassandra-env.sh
```

Comment:
the addition of `-Dcassandra.ring_delay_ms=0` makes Cassandra start up much quicker so the test can run and pass quicker.